languages/ocaml.json

```diff
@@ -25,7 +25,7 @@
   ],
   "indentationRules": {
     "decreaseIndentPattern": "^\\s*(end|done|with|in|else)\\b|^\\s*;;",
-    "increaseIndentPattern": "\\b(begin|object)\\s*$|\\blet [a-zA-Z0-9_-]+( [^ ]+)* =\\s*$|method!?[ \\t]+.*=[ \\t]*$|->[ \\t]*$|\\b(for|while)(_lwt)?[ \\t]+.*[ \\t]+do[ \\t]*$|(\\btry(_lwt)?$|\\bif\\s+.*\\sthen$|\\belse|[:=]\\s*sig|=\\s*struct)\\s*$",
+    "increaseIndentPattern": "\\b(begin|object)\\s*$|\\b(let|and)[\\*\\+]? [a-zA-Z0-9'_-]+( [^ ]+)* =\\s*$|method!?[ \\t]+.*=[ \\t]*$|->[ \\t]*$|\\b(for|while)(_lwt)?[ \\t]+.*[ \\t]+do[ \\t]*$|(\\btry(_lwt)?$|\\bif\\s+.*\\sthen$|\\belse|[:=]\\s*sig|=\\s*struct)\\s*$",
```

Comment:
Binding operators support more operators than just `+` and `*`.

You can use this regex from our OCaml grammar to support all of them: `([$&*+\\-/=>@^|<][!?$&*+\\-/=>@^|%:]*)`

https://github.com/ocamllabs/vscode-ocaml-platform/blob/a506da28c8c521ecf2033603850d084dc2c5e88b/syntaxes/ocaml.json#L379
https://v2.ocaml.org/releases/5.1/htmlman/bindingops.html
https://v2.ocaml.org/releases/5.1/htmlman/lex.html#core-operator-char
https://v2.ocaml.org/releases/5.1/htmlman/indexops.html#dot-operator-char

Reply:
Attempted in c705a6e5cde7451f0a3993522a5f0af231069f04